devtools-ci-index.yaml

```diff
@@ -563,6 +563,36 @@
               current-parameters: true
 
 - job:
+    name: 'eclipse-che-promotion'
+
+    parameters:
+        - string:
+            name: CHE_SERVER_STAGE_DOCKER_IMAGE
+            description: "Full docker location with organization/repo:tag"
+
+    wrappers:
+        - che_credentials_wrapper
+    defaults: global
+    node: devtools
+    builders:
+        - shell: |
+            # promoting to rhche
+            set +e
+            set +x
+
+            # downloads staging image
+            docker pull ${CHE_SERVER_STAGE_DOCKER_IMAGE}
+
+            # rename to production organization
+            PROD_DOCKER="${CHE_SERVER_STAGE_DOCKER_IMAGE/rhchestage/rhche}"
```

Comment:
Thanks for the updates @lordofthejars. One last thing: could you please rename `PROD_DOCKER` TO `CHE_SERVER_PROD_DOCKER_IMAGE`?